{{cookiecutter.repo_name}}/devtools/conda-envs/test_env.yaml

```diff
@@ -1,21 +1,28 @@
-name: test
-channels:
-{% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
-  - conda-forge
-{% endif %}
+name: {{cookiecutter.repo_name}}-test
+channels: {% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
+  - conda-forge {% endif %}
   - defaults
 dependencies:
     # Base depends
   - python
   - pip
 
+    # MDAnalysis
+  - cython
+  - numpy
+  {% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
+  - MDAnalysis
+  {% endif %}
+
     # Testing
   - pytest
   - pytest-cov
+  - pytest-xdist
 {% if cookiecutter.dependency_source == 'Prefer default anaconda channel with pip fallback' %}
   # Pip-only installs
   - pip:
     - codecov
+    - MDAnalysis
```

Comment:
for testing should we be pointing to develop? (maybe not in this file, but it's worth considering a runner that specifically installs the git head for develop,)

Reply:
Maybe there should just be a matrix of tests that does different MDA versions?

Reply:
we can make MDA a special case, IMO.

Reply:
This can be a separate PR.